public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -501,6 +489,51 @@ export class PanelModel implements DataConfigSource {
   }
 }
 
+function applyFieldConfigDefaults(fieldConfig: FieldConfigSource, defaults: FieldConfigSource): FieldConfigSource {
+  const result: FieldConfigSource = {
+    defaults: _.mergeWith(
+      {},
+      defaults.defaults,
+      fieldConfig ? fieldConfig.defaults : {},
+      (objValue: any, srcValue: any): any => {
+        if (_.isArray(srcValue)) {
+          return srcValue;
+        }
+      }
+    ),
+    overrides: [...defaults.overrides, ...(fieldConfig && fieldConfig.overrides ? fieldConfig.overrides : [])],
```

Comment:
I think this code is a mouthful and deserves some tests to explain it :)

Reply:
also thought this looked strange (this was just copied from the existing code that I extracted to this new method). 

Can simplify this as we do not actually support setting default overrides